docs/index.md

```diff
@@ -8,6 +8,13 @@ keywords: [Ethereum remote signer service, external vault support, HSM device su
 
 # Web3Signer
 
+:::caution YubiHSM 2 and USB Armory Mk II deprecation notice
+Web3Signer has deprecated private key storage support on USB Armory Mk II and YubiHSM 2, and will remove it in a future release.
+
+If you need this feature, consider maintaining a fork and submitting pull requests. Alternatively, you can
+use an older Web3Signer version that supports these storage mechanisms.
+:::
+
```

Comment:
Would this be better after this sentence in the next section? "Web3Signer can sign on multiple platforms using private keys stored in an external vault, or encrypted on a disk."